apps/server/src/modules/course/repo/coursegroup.repo.ts

```diff
@@ -30,4 +30,14 @@ export class CourseGroupRepo extends BaseRepo<CourseGroupEntity> {
 		});
 		return [courseGroups, count];
 	}
+
+	public async removeUserReference(userId: EntityId): Promise<number> {
+		const id = new ObjectId(userId);
+		const count = await this._em.nativeUpdate(this.entityName, { students: id }, {
+			// Note MikroOrm does not convert here students name to userId db field
```

Comment:
BTW, it should be possible to get the actual `fieldName` from the entity metadata, like
```typescript
import { MetadataStorage } from '@mikro-orm/core';

// Get the metadata for the entity
const metadata = MetadataStorage.getMetadata(CourseGroupEntity.name);

// Get the actual field name for the 'students' property
const fieldName = metadata.properties.students.fieldName; //  => 'userIds'
```
Maybe we could put that in a utility function. Or maybe there's already something in the MikroORM API.

Reply:
The key students do not work for native operations, because the decorator (change name in entit vs collection) to not work? Then i like to use solutions base on mikroORM metadata, otherwise it can break easy. But a point chain like .properties.students.fieldName can also break easy by and update.

Reply:
done, by adding  `getFieldName` helper function in shared folder